### PR TITLE
fix issue  #101

### LINKED ISSUE #101: Shell script to start Bassa servers at once
In the current project there is no script to start all the three Bassa servers at once. Therefore, writing a shell script to start Bassa servers at once will make users comfortable. I will send a PR shortly.

### PR DESCRIPTION
**Pull request by iammosespaulr**
Fix #101 
@malithsen @agentmilindu please review ..

### CODE REVIEW

**Comment by iammosespaulr** [comment]
@agentmilindu @malithsen Please review

File: start.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!bin/bash
+cd components/core;
+gnome-terminal -e "sudo python3 Main.py";
+cd ..;
+cd ..;
+cd ui/;
+gnome-terminal -e "sudo gulp serve";
```
Comment (agentmilindu) [inline start.sh]: Why `gnome-terminal`?

Comment (padamchopra) [reply start.sh]: @iammosespaulr I think that using this line should take care of both the servers:
```
sudo python3 components/core/Main.py & cd ui/ & sudo gulp serve;
```

**Comment by kmehant** [comment]
I don't think using `gnome-terminal` to execute this would be productive and is true that not all machines have it installed. I would really appreciate if you could come with a much more simple solution to start all the servers. So closing this **PR** for now.

**Pull request closed without merging**